localstack/services/lambda_/resource_providers/aws_lambda_eventsourcemapping.py

```diff
@@ -180,4 +180,36 @@ def update(
           - lambda:UpdateEventSourceMapping
           - lambda:GetEventSourceMapping
         """
-        raise NotImplementedError
```

Comment:
Glad to see some more updates supported, especially for highly demanded functionality 📈